### PR TITLE
Add prescription refresh workflow for image analysis handler

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

Related-To: https://github.com/AICoE/recommend-base-image-tutorial/issues/1
Fixes: https://github.com/thoth-station/thoth-application/issues/2229

### CODE REVIEW

**Review by fridex — APPROVED** [review]
lgtm! 👍🏻 🥇

**Comment by goern** [comment]
looks nice to me, can this go into next release?
/milestone 2022.01.18

File: prescriptions-refresh-job/overlays/ocp4-stage/imagestreamtag.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: prescriptions-refresh-job
+spec:
+  tags:
+    - name: latest
+      from:
+        kind: DockerImage
+        name: quay.io/thoth-station/prescriptions-refresh-job:kebechet-v0.5.0
```
Comment (harshad16) [inline prescriptions-refresh-job/overlays/ocp4-stage/imagestreamtag.yaml:11]:
```suggestion
        name: quay.io/thoth-station/prescriptions-refresh-job:v0.5.0
```
This image is now available

File: prescriptions-refresh-job/overlays/ocp4-stage/thoth-notification.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        - name: chat-notification
+          image: registry.access.redhat.com/ubi8/ubi
+          command:
+            - "curl"
+            - "-X"
+            - "POST"
+            - "-H"
+            - "Content-Type: application/json; charset=UTF-8"
+            - "-d"
+            - "{'text':'Successfully synchronized *prescriptions-refresh-job* to *OCP4-STAGE*, see <https://argocd.operate-first.cloud/applications/prod-thoth-prescriptions-refresh-job|ArgoCD UI> 🚚'}"
```
Comment (harshad16) [inline prescriptions-refresh-job/overlays/ocp4-stage/thoth-notification.yaml:25]:
```suggestion
            - "{'text':'Successfully synchronized *prescriptions-refresh-job* to *OCP4-STAGE*, see <https://argocd.operate-first.cloud/applications/stage-thoth-prescriptions-refresh-job|ArgoCD UI> 🚚'}"
```

**Review by harshad16 — CHANGES_REQUESTED** [review]
lgtm
just few minor chnages required.

**Review by harshad16 — APPROVED** [review]
/unhold
thanks :100:

**Pull request merged**